skimage/transform/integral.py

```diff
@@ -1,6 +1,8 @@
 import numpy as np
 import collections
 import warnings
+from .._shared.kahan import kahan_cumsum
+
 
 def integral_image(img):
```

Comment:
what is the performance cost of Kahan summation? If it's significant, could ``integral_image`` have a keyword argument ``accurate=False`` that would call Kahan summation if ``True``?